napari/layers/labels/labels.py

```diff
@@ -818,22 +818,7 @@ def _get_color_lookup_func(self, data, min_label_val, max_label_val):
                 self._all_vals[0] = 0
             return self._lookup_with_index
 
-    def _raw_to_displayed(self, raw):
-        """Determine displayed image from a saved raw image and a saved seed.
-
-        This function ensures that the 0 label gets mapped to the 0 displayed
-        pixel.
-
-        Parameters
-        ----------
-        raw : array or int
-            Raw integer input image.
-
-        Returns
-        -------
-        image : array
-            Image mapped between 0 and 1 to be displayed.
-        """
+    def _raw_to_displayed_choose_coloring(self, raw):
```

Comment:
is there a reason to split this out into a new function `_raw_to_displayed_choose_coloring`? it looks like it only gets called in once place. I find the previous version slightly more readable. It will also make the diff here smaller and easier to compare

Reply:
For me, this function is to long to be readable, but I revert the split (also during development there was more than one place of call)